packages/lodestar/src/chain/blocks/importBlock.ts

```diff
@@ -118,12 +123,37 @@ export async function importBlock(chain: ImportBlockModules, fullyVerifiedBlock:
       throw Error(`merge block parent's parent POW block not found ${powBlock.parentHash}`);
     onBlockPrecachedData.powBlock = powBlock;
     onBlockPrecachedData.powBlockParent = powBlockParent;
-    onBlockPrecachedData.isMergeTransitionBlock = true;
   }
 
   const prevFinalizedEpoch = chain.forkChoice.getFinalizedCheckpoint().epoch;
+  const prevIsMergeTransitionFinalized = chain.forkChoice.getMergeTransitionFinalized();
+
   chain.forkChoice.onBlock(block.message, postState, onBlockPrecachedData);
 
+  // If this was a merge transition block, it has been successfully imported
+  // in forkchoice! Its now PoS all the way!!!!
+  if (onBlockPrecachedData.isMergeTransitionBlock) {
+    const mergeBlock = block.message as bellatrix.BeaconBlock;
+    const mergeBlockHash = toHexString(ssz.bellatrix.BeaconBlock.hashTreeRoot(mergeBlock));
+    const mergeExecutionHash = toHexString(mergeBlock.body.executionPayload.blockHash);
+    const mergePowHash = toHexString(mergeBlock.body.executionPayload.parentHash);
+    chain.logger.info(POS_PANDA_TRANSITIONING_BANNER);
+    chain.logger.info("Execution transitioning from PoW to PoS!!!");
+    chain.logger.info("Imported terminal PoW block", {
+      blockHash: mergeBlockHash,
+      executionHash: mergeExecutionHash,
+      powHash: mergePowHash,
+    });
+  }
+  if (!prevIsMergeTransitionFinalized && chain.forkChoice.getMergeTransitionFinalized()) {
+    const finalizedCheckpoint = chain.forkChoice.getFinalizedCheckpoint();
+    chain.logger.info(POS_PANDA_FINALIZED_BANNER);
```

Comment:
@philknows yea this banner can go, or it can be super minimal, or we can add it later now we have a spot